doc/Development-Guide-Rad-Tool.md

```diff
@@ -3,43 +3,46 @@
+DLLs (that are inside the folder mentioned above) do all the work. The extension is just a user interface. This design is required, otherwise it would only be available for Visual Studio Windows users. Howver, since the tool is built on .NET Core platform, **Mac** or **Linux** users can safely use the tool. 
+
+On Mac and Linux, you have to manually do some of the work that is done by the extension. This involves just creating a short and basic[ JSON file] as input (https://aspnetzero.com/Documents/Development-Guide-Rad-Tool-Mac-Linux).
 
 ### How To Edit Pre-defined Templates Or Create A New Template?
 
- The templates are inside "/AspNetZeroRadTool/FileTemplates" directory in your project's root directory. Every template is splitted into 3 file ("MainTemplate.txt", "PartialTemplates.txt","TemplateInfo.txt") . If you want to edit any file, just replicate it in same directory and change it's extension to ".custom.txt" from ".txt".  For example, you can create "MainTemplate.custom.txt" to override "MainTemplate.txt" in same directory. Plase don't make any change on orginal templates.
- To create a new template, do it the way we do in pre-defined templates. Tool doesn't know any info about templates and explore them in "/FileTemplates" directory every time it is run. So your new template will be precessed like pre-defined ones. (".custom" extension is not needed for new templates.)
+ The templates are inside "/AspNetZeroRadTool/FileTemplates" directory in your project's root directory. Every template is split into three files: "MainTemplate.txt", "PartialTemplates.txt" and "TemplateInfo.txt". If you want to edit any file, just copy it in same directory and change it's extension to ".custom.txt" from ".txt".   For example, you can create "MainTemplate.custom.txt" to override "MainTemplate.txt" in same directory. Plase don't make any changes to the orginal templates.
+ 
+ To create a new template, do the same process as the editing a pre-defined templates. The tool doesn't know any info about templates and discovers them in the "/FileTemplates" directory every time it is run.  This means your new template will be processed like the pre-defined ones. (".custom" extension is not needed for new templates.)
 
- You can request help from our support team on [github](github.com/aspnetzero/aspnet-zero-core)  or  [forum](https://forum.aspnetboilerplate.com/viewforum.php?f=5)  if you are struggling
+ You can request help from our support team on [github](github.com/aspnetzero/aspnet-zero-core)  or  [forum](https://forum.aspnetboilerplate.com/viewforum.php?f=5)  if you are struggling.
 
 
 ### Generated Files
 
- Here is the full list of the files that are created or modified by the tool, if you give a basic "Cars" entity as input.
+ Here is the full list of the files that are created or modified by the tool assuming you give a basic "Cars" entity as input.
```

Comment:
The change from `, if` to `assuming` is not appropriate.

Reply:
There really shouldn't be a comma here (https://www.grammarly.com/blog/comma/).  If anything, the conditional clause at the end should be wrapped in parenthesis or joined with a word like 'assuming'.